.github/workflows/publish-image.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish images to ghcr
+
+on:
+  push:
+    branches: ['master']
```

Comment:
I lke _building_ the Docker image on each push to `master` but pushing images could lead to confusion about which version is being used, or using something that is prerelease. It would probably be better to push images when a tag is created (and then tag the image with the same version number as the tag).